models/marts/fct_orders.sql

```diff
@@ -0,0 +1,23 @@
+-- note there could be 2 payments for one order
+
+{{ config(materialized='table') }}
+
+
+WITH stg_orders AS (
+    SELECT *
+    FROM {{ ref('stg_orders') }}
+),
+stg_payments AS (
+    SELECT *
+    FROM {{ ref('stg_payments') }}
+),
```

Comment:
Related to your note above, in this model, you might want to handle the multiple payments here!